T-Sec.LinuxOS.Compliance/tasks/hardening_linux(02)logging.yml

```diff
@@ -114,32 +120,6 @@
   notify: restart auditd
   when: config_auditd
 
-- name: req-035.2 search for privileged commands
-  shell: df --local -P | awk {'if (NR!=1) print $6'} | xargs -I '{}' find '{}' -xdev -type f \( -perm -4000 -o -perm -2000 \) -print 2>/dev/null
-  register: priv_commands
-  changed_when: false
-  when: config_auditd and config_access_events
-
-- name: req-035.3 configure logging for priviledged commands
-  lineinfile:
-    path: "{{ audit_rules_file }}"
-    insertafter: EOF
-    line: '-a always,exit -F path={{ item }} -F perm=x -F auid>=1000 -F auid!=4294967295 -k privileged'
-    state: present
-  with_items: '{{ priv_commands.stdout_lines }}'
-  notify: restart auditd
-  when: config_auditd and config_access_events
-
-# Req 38:	Auditd configuration must be immutable.
-
-- name: req-038.1 make audit configuration immutable
```

Comment:
I'm wondering how to refer to `req-038.1 add a comment in the jinja2 at least that it refers to that requirement` - taking it out without reference is not a good idea. Since it has been moved to the jinja2 template, maybe a placemark task with print/debug/msg && comment in the template?